docker/centos7/Dockerfile

```diff
@@ -56,6 +56,12 @@ RUN yum install -y \
 # Our scripts assume that cmake is called 'cmake'
 RUN ln -s /usr/bin/cmake3 /usr/bin/cmake
 
+# Install gcc-9
+RUN yum install -y centos-release-scl
+RUN yum install -y devtoolset-9
+RUN echo "source /opt/rh/devtoolset-9/enable" >> /etc/bashrc
+SHELL ["/bin/bash", "--login", "-c"]
```

Comment:
The previous images are less than 0.5GiB, but this PR seems to generate `1.8GB` image.
![Screenshot 2022-11-28 at 6 23 11 PM](https://user-images.githubusercontent.com/9700541/204423215-b1ad412a-c354-4a47-9db1-28886e686e6b.png)

```
$ docker images | head
REPOSITORY                                                 TAG                                                                          IMAGE ID       CREATED         SIZE
apache/orc-dev                                             centos7                                                                      d2dc4ee767e0   5 minutes ago   1.8GB
```

Reply:
That's probably from the dependency of devtoolset-9